recipes/configs/mistral/7B_lora.yaml

```diff
@@ -71,3 +71,7 @@ metric_logger:
   log_dir: ${output_dir}
 output_dir: /tmp/Mistral-7B-v0.1
 log_every_n_steps: null
+
+# Pytorch Profiling
+enable_torch_profiler: False
```

Comment:
Not sure on thoughts of how scalable adding flags to every single recipe is... @RdoubleA any thoughts?